app/assets/stylesheets/components/_action-link.scss

```diff
@@ -0,0 +1,100 @@
+.app-c-action-link {
+  display: inline-block;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-align: center;
+  align-items: center;
+  box-sizing: border-box;
+  padding-left: 50px;
+  min-height: 45px;
+  max-width: 460px;
+  background-image: image-url("covid-arrow.png"); // PNG fallback for SVG
+  background-image: image-url("covid-arrow.svg");
+  background-repeat: no-repeat;
+  background-position: 0 50%;
+  background-size: 37px auto;
+
+  @include govuk-media-query($until: tablet) {
+    max-width: 410px;
+    margin-bottom: govuk-spacing(8);
+  }
+
+  @include govuk-media-query($from: desktop) {
+    // IE10/11 specific - vertical align in flexbox is ignored
+    // add padding to action link container to improve arrow/text alignment on IE
+    @media screen and (min-width:0\0) {
+      padding-top: govuk-spacing(2);
+    }
+  }
+}
+
+.app-c-action-link__nowrap-text {
+  white-space: nowrap;
+}
+
+.app-c-action-link__link {
+  @include govuk-font(19, $weight: bold);
+  display: inline-block;
+  max-width: 100%;
+
+  &:hover {
+    text-decoration: underline;
+  }
+
+  &:focus,
+  &:active {
+    text-decoration: none;
+    color: govuk-colour("black");
+  }
+}
+
+.app-c-action-link--dark-icon {
+  @include govuk-media-query($until: tablet) {
+    margin-bottom: govuk-spacing(2);
+  }
+
+  @include govuk-media-query($from: desktop) {
+    // IE10/11 specific - vertical align in flexbox is ignored
+    @media screen and (min-width:0\0) {
+      padding-top: 0;
+    }
+  }
+
+  background-image: image-url("corona-arrow-mini-dark.png"); // PNG fallback for SVG
+  background-image: image-url("corona-arrow-mini-dark.svg");
+  background-size: 25px auto;
+  background-position: 0 0;
```

Comment:
Just curious - if this line is removed (and it falls back to `0 50%` from the original class) does it not work?

Reply:
It does work, however it would be slightly off-centre and would not look the same as the current live site version. 

Current version `background-position: 0 0` one line/multiline
![0-oneline](https://user-images.githubusercontent.com/7116819/80117979-bb6c5000-857f-11ea-85ac-8db416c24402.png)
![0-multiline](https://user-images.githubusercontent.com/7116819/80117972-ba3b2300-857f-11ea-8e0c-44d13314ca76.png)

If `background-position: 0 50%` one line/multiline
![50-oneline](https://user-images.githubusercontent.com/7116819/80117983-bc04e680-857f-11ea-8796-cba2be6c2a32.png)
![50-multiline](https://user-images.githubusercontent.com/7116819/80117981-bc04e680-857f-11ea-89ca-4268a5a8732c.png)

Reply:
That's fine, don't worry then.